paddle/operators/fill_constant_batch_size_like_op.cc

```diff
@@ -100,5 +100,5 @@ REGISTER_OPERATOR(fill_constant_batch_size_like,
                   ops::FillConstantBatchSizeLikeOpMaker);
 REGISTER_OP_CPU_KERNEL(
     fill_constant_batch_size_like,
-    ops::FillConstantBatchSizeLikeOpKernel<paddle::platform::CPUPlace, float>,
-    ops::FillConstantBatchSizeLikeOpKernel<paddle::platform::CPUPlace, double>);
+    ops::FillConstantOpKernel<paddle::platform::CPUPlace, float>,
+    ops::FillConstantOpKernel<paddle::platform::CPUPlace, double>);
```

Comment:
Cool!

Reply:
@tonyyang-svail Sorry, since https://github.com/PaddlePaddle/Paddle/pull/5441 change the implementation of `FillConstantOpKernel`, adding `force_cpu` for this kernel. I resume this file.